borg/archiver.py

```diff
@@ -1919,12 +1925,26 @@ def parse_args(self, args=None):
         update_excludes(args)
         return args
 
+    def _setup_implied_logging(self, args):
+        """ turn on INFO level logging for args that imply that they will produce output """
+        # map of option name to name of child logger (which matches the option name)
+        option_logger = {
+                'output_list': 'list',
+                'show_version': 'show-version',
+                'show_rc': 'show-rc',
+                'stats': 'stats',
```

Comment:
Yes.  I actually had to rework how the alternate logger was selected, since that made using child loggers of the current logger unworkable.  See the d163621 commit message for more detail.